src/interface/app.py

```diff
@@ -0,0 +1,31 @@
+import streamlit as st
+
+from typing import List
+from .css import load_css
+from .gui import Gui
+
+
+def create_gui(path_file: str,
+               path_modified_file: str,
+               message_container_height: int,
+               layout_division: List[int]) -> None:
+
+    if "application" not in st.session_state:
+        st.session_state.application = Gui(path_file, path_modified_file, message_container_height)
+
+    load_css()
```

Comment:
This streamlit code should be all inside the GUI class. We don't want to have it splitted into separate function and class.  That's why we have to think how to organize it.